packages/main-api/src/controllers/m1/bills/mod.rs

```diff
@@ -0,0 +1,58 @@
+#[allow(unused_imports)]
+use crate::utils::openapi::*;
+use bdk::prelude::*;
+use by_axum::axum::{extract::State, routing::post};
+use dto::*;
+
+const BATCH_SIZE: u32 = 10;
+
+#[derive(Clone, Debug)]
+pub struct BillsControllerM1 {
```

Comment:
```suggestion
pub struct BillsController {
```

remove version because the module path already contains version.